isis/tests/ClipperWacFcCameraTests.cpp

```diff
@@ -66,4 +66,33 @@ TEST_F(ClipperWacFcCube, ClipperWacFcCameraUnitTest) {
   std::pair<iTime, iTime> startStop;
   startStop = cam->ShutterOpenCloseTimes(etStart.Et(), 0.00005);  // dummy value for exposure duration
   EXPECT_TRUE(startStop.first.Et() < startStop.second.Et());
+
+
+  EXPECT_TRUE(cam->SetImage(145, 161));
+  EXPECT_DOUBLE_EQ(cam->UniversalLatitude(), 8.5881709286625423);
```

Comment:
Can this be accomplished with a change to ```TestLineSamp(Camera *cam, double samp, double line)``` ?.
i.e.```TestLineSamp(Camera *cam, double samp, double line, double lat, double lon)```
Since we expect test data to change it would make the tests simpler to update